docs/reference/search/search.asciidoc

```diff
@@ -214,35 +218,184 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=timeout]
   <<query-dsl,Query DSL>>.
 
 
+[[search-api-response-body]]
+==== {api-response-body-title}
+
+`took`::
++
+--
+(Integer)
+Milliseconds it took {es} to execute the request.
+
+This value is calculated by measuring the time elapsed
+between receipt of a request
+and its execution.
+
+Took time does *not* include:
+
+* Serializing a request into JSON on a client
+* Sending a request over a network
+* Deserializing a request from JSON on a server
+* Serializing a response into JSON on a server
+* Sending a response over a network
+* Deserializing a response from JSON on a client
+--
```

Comment:
I am not totally sure about this part. In the question that you linked the user was asking about discrepancy between the time that he was measuring and the time that we reported in took. That's why I concentrated in my answer on the things that happen outside of elasticsearch. I feel that in the documentation we might want to concentrate on stuff that happens inside elasticsearch. So, for documentation I would probably rephrase it  as something like this: 

This value is calculated by measuring the time elapsed between receipt of a request on the coordinating node and the time the coordinating node is ready to send the response. It includes communication time between the coordinating node and data nodes, the time the request is waiting in the thread pool search queue to be executed and the actual execution time. It doesn't include time that is required to send a request to elasticsearch as well as sending a response back to a client.

Reply:
Thanks for the feedback.

I've amended the `took` description with 1ac4bd65a9b4c1c2d2d6d76003c87b051e5b0573.